packages/sample-app-fabric/windows/SampleAppFabric/SampleAppFabric.cpp

```diff
@@ -102,10 +62,45 @@ _Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR
   auto hwnd = winrt::Microsoft::UI::GetWindowFromWindowId(window.Id());
   auto scaleFactor = ScaleFactor(hwnd);
 
-  auto host = CreateReactNativeHost(hwnd, compositor);
+  WCHAR appDirectory[MAX_PATH];
+  GetModuleFileNameW(NULL, appDirectory, MAX_PATH);
+  PathCchRemoveFileSpec(appDirectory, MAX_PATH);
+
+  auto reactInstanceSettingsBuilder {
+    winrt::Microsoft::ReactNative::ReactInstanceSettingsBuilder()
+#if BUNDLE
+        .JavaScriptBundleFile(L"index.windows")
```

Comment:
We should be able to have JavaScriptBundleFile be the same for both bundle and non-bundle.  And similarly, we should always set BundleRootPath.

The only thing difference in settings we should be doing with #if BUNDLE is setting the default value of UseFastRefresh.

The dev can toggle UseFastRefresh through the dev menu, so if the other settings line up properly, then they should be able to switch between the prebuilt bundle and metro bundle without having to do any native rebuilds.

Reply:
Addressed this feedback. Thanks.